livekit_room.proto

```diff
@@ -141,3 +144,9 @@ message SendDataRequest {
 message SendDataResponse {
   //
 }
+
+message UpdateRoomRequest {
+  string room = 1;
```

Comment:
@FeepsDev I wasn't sure if we'd need an identity field here as well? I left it out for now, but start to think that was a mistake...

Reply:
I don't think identity is needed. it is a room-level request.